components/config-mgmt-service/params/parameters.go

```diff
@@ -50,6 +50,8 @@ func ConvertParamToNodeRunBackend(parameter string) string {
 		return backend.ChefVersion
 	case "chef_tags":
 		return backend.ChefTags
+	case "error":
+		return "error_message"
```

Comment:
It looks like the rest of these things are defined as constants, should `error_message` get moved over there?